lib/new-utils.js

```diff
@@ -2,23 +2,34 @@
 
 const Long = require('long');
 
+const maxInt = BigInt(0x100000000);
```

Comment:
Is it really the maximum possible Int? Isn't e.g. `0x100000001` possible?

Reply:
This is reversed engineered from how the Long type keeps it's values split between high and low

Reply:
OK, good. Then please explain this in a comment.